Vagrantfile

```diff
@@ -124,19 +135,12 @@ Vagrant.configure("2") do |global_config|
         v.tags = {'Name' => 'swift'}
       end
 
-      if vagrant_box != 'jammy' then
-        # Install libssl for Chef (https://github.com/hashicorp/vagrant/issues/10914)
-        config.vm.provision "shell",
-          inline: "sudo apt-get update -y -qq && "\
-            "export DEBIAN_FRONTEND=noninteractive && "\
-            "sudo -E apt-get -q --option \"Dpkg::Options::=--force-confold\" --assume-yes install libssl1.1"
-      end
-
```

Comment:
Isn't this code still required for other boxes/providers? Can we just update the `if` to excluide for AWS. Maybe if vagrant_box is "dummy" or VAGRANT_DEFAULT_PROVIDER == aws?

```
unless ['jammy', 'dummy'].include? vagrant_box then
 ....
```
Or maybe:
```
unless vagrant_box.include? "jammy" || ENV['VAGRANT_DEFAULT_PROVIDER'] == 'aws' then
...
```

NOTE I haven't actaully tested any of that code :)